controllers/v1/programs.js

```
@@ -737,4 +738,27 @@ module.exports = class Programs extends Abstract {
+			try {
+				// Call helper method to perform the $pull operation on programs collection
+				let programData = await programsHelper.removeSolutionsFromProgramComponent(req.params._id)
+				return resolve(programData)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)